server/pulp/server/controllers/repository.py

```diff
@@ -293,15 +294,13 @@ def delete(repo_id):
     # the server logs for more information.
     error_tuples = []  # tuple of failed step and exception arguments
 
-    importer_manager = manager_factory.repo_importer_manager()
     distributor_manager = manager_factory.repo_distributor_manager()
 
     # Inform the importer
-    importer_coll = RepoImporter.get_collection()
-    repo_importer = importer_coll.find_one({'repo_id': repo_id})
+    repo_importer = model.Importer.objects(repo_id=repo_id).first()
```

Comment:
same question as elsewhere: can you use ``.get()`` instead, which ensures only one is found?